src/StorageStack.ts.orig

```diff
@@ -0,0 +1,144 @@
+import { Duration, Stack, StackProps } from 'aws-cdk-lib';
+import { AttributeType, BillingMode, Table, TableEncryption } from 'aws-cdk-lib/aws-dynamodb';
+import { Effect, PolicyStatement } from 'aws-cdk-lib/aws-iam';
+import { Key } from 'aws-cdk-lib/aws-kms';
+import { Bucket, BucketEncryption, ObjectOwnership } from 'aws-cdk-lib/aws-s3';
+import { Secret } from 'aws-cdk-lib/aws-secretsmanager';
+import { StringParameter } from 'aws-cdk-lib/aws-ssm';
+import { Construct } from 'constructs';
+import { Configurable } from './Configuration';
+import { Statics } from './statics';
+
+
+interface StorageStackProps extends StackProps, Configurable {};
+
+/**
+ * Contains all API-related resources.
+ */
+export class StorageStack extends Stack {
+  constructor(scope: Construct, id: string, props: StorageStackProps) {
+    super(scope, id, props);
+
+    const key = this.key();
+    /**
+     * This bucket will receive submission attachments
+     * (Submission PDF, uploads) for each submission.
+     */
+    const bucket = new Bucket(this, 'submission-attachments', {
+      eventBridgeEnabled: true,
+      enforceSSL: true,
+      encryption: BucketEncryption.KMS,
+      objectOwnership: ObjectOwnership.BUCKET_OWNER_ENFORCED,
+      lifecycleRules: [
+        {
+          expiration: Duration.days(365),
```

Comment:
waarom 365 dagen bewaren en niet korter of langer(zou ik verwachten nl)?

Reply:
Daar heeft nog niemand iets van gevonden, onbeperkt lijkt me ook onwenselijk. Ik zie overigens dat de merge conflict-file hierin zit, dus dit is niet nieuw.